src/main/target/common_pre.h

```diff
@@ -48,7 +48,6 @@
 #endif
 
 #ifdef STM32F4
-#define USE_SRAM2
```

Comment:
Since we are pulling this I think the conditional below that defines `SRAM2` can be removed as well - it is not used anywhere.

Reply:
OK. All definitions of USE_SRAM2 are gone.